.github/workflows/ci.yml

```
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
+    - uses: actions/setup-node@v3
       with:
-        node-version: 12.x
+        node-version: 16
```

I'm pretty sure that just `16` will get you 16.0.0

```suggestion
        node-version: 16.x
```